tests/Unit/NumericalAlgorithms/RootFinding/Test_NewtonRaphson.cpp

```diff
@@ -0,0 +1,140 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <cmath>
+#include <cstddef>
+#include <limits>
+#include <utility>
+
+#include "DataStructures/DataVector.hpp"
+#include "ErrorHandling/Error.hpp"
+#include "NumericalAlgorithms/RootFinding/NewtonRaphson.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "tests/Unit/TestingFramework.hpp"
```

Comment:
should be first include (i.e. right after the license) (see https://spectre-code.org/writing_unit_tests.html)